### ISSUE TITLE
The method CellsUsed without parameters returns cellls which have only format

### ISSUE THREAD

**Issue by traf72** [ClosedXML/ClosedXML#1397]
## Read and complete the full issue template

**Do you want to request a *feature* or report a *bug*?**
- [x] Bug
- [ ] Feature

**Version of ClosedXML**

e.g. 0.95.0

**What is the current behavior?**

The method CellsUsed without parameters returns cellls which have only format.

**What is the expected behavior or new feature?**

The method CellsUsed without parameters returns cellls with content only.

**Did this work in previous versions of our tool?  Which versions?**

Yes/v0.94.0

## Reproducibility

**Code to reproduce problem:**
```c#
public void Main()
{
    var wb = new XLWorkbook();
    var ws = wb.AddWorksheet("Test");
    ws.Cell(1, 1).Value = "Value";
    ws.Cell(1, 2).Style.Border.OutsideBorder = XLBorderStyleValues.Thin;
    var cellsUsedCount = ws.CellsUsed().Count(); // returns 2, but must be 1
}
```

**Comment by igitur** [comment]
Fixed in #1399

**Comment by igitur** [comment]
v0.95.1 has been deployed to NuGet. Please test and let me know if this solves your problem.
